### DISCUSSION TITLE
I can login, but my profile and contributions aren't visible to others

### DISCUSSION THREAD

**Discussion by hellocloudnative** [community/community#114555] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, Mr. or Ms.

I don’t understand what happened to my account. Would you tell me what to do now? Thank you very much for your support. I don’t know why my account has been flagged.

**Comment by hellocloudnative** [comment]
help！

**Reply by ksdh2333** [reply]
I encountered the same situation, can you tell me if you have already solved it through some method?

**Comment by jcduro** [comment]
Siento que tengas este problema. Hay varias razones por las que tu perfil y contribuciones pueden no ser visibles. Aquí están los pasos para resolver esto:

**1. Verifica tus configuraciones de privacidad:**
- Ve a Settings > Profile
- Asegúrate de que "Make my profile private" esté desactivado
- Confirma que "Make my contributions public" esté habilitado

**2. Revisa la restricción de cuenta:**
- Si recibiste un mensaje sobre una violación de términos, tu perfil puede estar limitado
- Verifica tu email para notificaciones de GitHub sobre restricciones de cuenta
- Lee el correo cuidadosamente - puede haber un formulario de apelación

**3. Comprueba el estado de tu email:**
- Ve a Settings > Emails
- Asegúrate de que tu email principal esté verificado (checkmark verde)
- Si no está verificado, revisa tu bandeja de spam

**4. Limpia el caché del navegador:**
- Presiona Ctrl+Shift+Del
- Borra cookies y caché de GitHub
- Cierra sesión completamente
- Vuelve a iniciar sesión

**5. Si nada funciona:**
- Contacta a GitHub Support en https://support.github.com/
- Proporciona detalles sobre cuándo comenzó el problema
- Incluye un screenshot de tus configuraciones

¿Revisaste alguna de estas opciones? ¿Tu cuenta está restringida o simplemente no visible?

**Comment by ZoroUchiha35** [comment]
I know it's been like a year but I can still help you with that, it's usually just updating this and that's all. Just go to settings and scroll down under Public Profile and click this 3 updates and evrything will be OK, if it doesn't work then the reason is you created an account that's private which is only to you who can see
<img width="1037" height="607" alt="image" src="https://github.com/user-attachments/assets/d149ee2d-bd03-4d03-91e4-83854826f309" />

**Reply by Spongebob-Uchiha** [reply]
You got a point
